src/applications/applications.service.ts

```
@@ -413,60 +413,86 @@ private resolveDynamicFields(
+
+					debugInfo.logicUsed = condition;
+
+					if (matches) {
+						if (condition.then.amount === 'value') {
+							amount = Number(applicationData[rule.inputFields[0]]) || 0;
+						} else {
+							amount = Number(condition.then.amount) || 0;
+						}
+						debugInfo.resolvedValues = {
+							inputValue: applicationData[rule.inputFields[0]],
+							then: condition.then,
```

Biome (1.9.4)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)